src/delay.rs

```diff
@@ -171,15 +172,14 @@ impl embedded_hal::delay::DelayNs for Ets {
 pub struct FreeRtos;
 
 impl FreeRtos {
-    #[deprecated = "Use delay_ns instead"]
     pub fn delay_us(us: u32) {
```

Comment:
So if I understand your changes correctly, you have basically changed the rounding from `floor` type to `accounting` type (i.e. "if larger than 0.5 round to the ceiling else round to the floor").

What I don't understand is two things:
* How exactly this is is fixing the "delays were way too long **(many orders of magnitude)**" (emphasis mine). I mean, if you requested from e.g. `Ets` a delay of  1.6 ns, previously it would delay by ~ 1ns. Now it would delay ~ 2ns. How is this "many orders of magnitude". Maybe I do not interpret your changes correctly?
* What was the difference between "latest GIT" and the released version? I don't think *anything* has changed in terms of how delays are computed, during the last ~3-4 months, if not longer?